app/screens/BookDetail/components/BookDetailOptions/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { View, Text, Image } from 'react-native';
+
+import defaultImg from '../../../../../assets/no_image.jpeg';
+import Button from '../../../../components/baseComponents/Button';
+import { lightBlue, white } from '../../../../../scss/colors';
+
+import { styles } from './style';
+
+function BookDetailOptions({ data: { title, author, year, genre, image_url: imageUrl } }) {
```

Comment:
if `data` is empty, this destructuring will  crash the app. I'd set some kind of default so that this doesn't happen.